bundles/SyliusThemeBundle/theme_configuration_reference.rst

```diff
@@ -0,0 +1,14 @@
+Theme configuration reference
+=============================
+
+.. code-block:: json
+
+    {
+        "name": "Theme human-readable name",
+        "logical_name": "vendor/sylius-theme",
```

Comment:
That was some convention in templating that uses `getLogicalName()` methods and we're decorating it with `logical_name` from the configuration.